types/rlp_unmarshal_storage.go

```diff
@@ -93,20 +133,45 @@ func (r *Receipts) UnmarshalStoreRLPFrom(p *fastrlp.Parser, v *fastrlp.Value) er
 		return err
 	}
 
-	for _, elem := range elems {
+	for i := 0; i < len(elems); i++ {
```

Comment:
Looks like dup code with `rlp_unmarshal.go`, a func woul be great.

Reply:
They are similar but use different objects. Generalising this func would look overcomplicated IMO. Feel free to suggest something if you want.